__integrations__/docker-compose.yml

```diff
@@ -0,0 +1,60 @@
+version: "3.2"
+
+services:
+
+  functions-node-18:
+    image: functions-node-18
+    build:
+      context: ..
+      dockerfile: ./docker/functions-node-18/Dockerfile
+    command: /bin/true
+
+  cosmosdb:
+    image: cosmosdb
+    env_file:
+      - ./environments/generated/env.cosmosdb
+    build:
+      context: ../docker/cosmosdb
+      dockerfile: ./Dockerfile
+    ports:
+      - ${COSMOSDB_PORT}:3000
+
+  storage-account:
+    image: azurite
+    build:
+      context: ../docker/azurite
+      dockerfile: ./Dockerfile
+    ports:
+      - 10000:10000
+      - 10001:10001
+      - 10002:10002
+
+  function:
+    image: fn
+    env_file:
+      - ./environments/generated/env.function
+    build:
+      context: ..
+      dockerfile: ./docker/functions/Dockerfile
+    ports:
+      - ${FUNCTION_PORT}:7071
+    links:
+      - cosmosdb
+      - storage-account
+    depends_on:
+      - functions-node-18
+
+  testagent:
+    image: node:16-alpine
```

Comment:
why node 16 here?

Reply:
This is the image used for calling tests, so it's not a problem. However I'll updated it, thanks.

Reply:
Done in [[#IOCIT-357] update node-version of it tests image](https://github.com/pagopa/io-functions-lollipop/pull/9/commits/2534de5eaa8695fbb08181c6e739534740b235ea)